src/Microsoft.VisualStudio.SlowCheetah.VS/Package/PreviewTransformCommand.cs

```diff
@@ -252,13 +252,13 @@ private void PreviewTransform(IVsHierarchy hier, string sourceFile, string trans
                     }
                     else if (!File.Exists(advancedOptionsPage.PreviewToolExecutablePath))
                     {
-                        throw new FileNotFoundException(string.Format(Resources.Resources.Error_CantFindPreviewTool, advancedOptionsPage.PreviewToolExecutablePath), advancedOptionsPage.PreviewToolExecutablePath);
+                        throw new FileNotFoundException(string.Format(CultureInfo.CurrentCulture, Resources.Resources.Error_CantFindPreviewTool, advancedOptionsPage.PreviewToolExecutablePath), advancedOptionsPage.PreviewToolExecutablePath);
                     }
                     else
                     {
                         // Open a process with the specified diff tool
                         // Add quotes to the file names
-                        ProcessStartInfo psi = new ProcessStartInfo(advancedOptionsPage.PreviewToolExecutablePath, string.Format(advancedOptionsPage.PreviewToolCommandLine, $"\"{sourceFile}\"", $"\"{destFile}\""))
+                        ProcessStartInfo psi = new ProcessStartInfo(advancedOptionsPage.PreviewToolExecutablePath, string.Format(CultureInfo.CurrentCulture, advancedOptionsPage.PreviewToolCommandLine, $"\"{sourceFile}\"", $"\"{destFile}\""))
```

Comment:
You should have the quotes around `sourceFile` and `destFile` inside the `PreviewToolCommandLine` resource itself.